@hubblecommerce/hubble/core/components/detail/ProductDetailCrosssellings.vue

```diff
@@ -66,10 +72,15 @@ export default {
     },
 
     async mounted() {
-        try {
-            this.crossSellings = await this.fetchCrossSeelings();
-        } catch (e) {
-            console.log(e);
+        if (!this.crossSellings && this.productId !== '') {
+            try {
+                const xSellResponse = await this.fetchCrossSeelings();
+                this.items = xSellResponse.data;
```

Comment:
Keep consistence in naming: xSellResponse => crossSellingResponse.
Also fix typo: fetchCrossSeelings => fetchCrossSelling

Reply:
Fixed in [c1daf44](https://github.com/hubblecommerce/hubble-frontend-pwa/commit/c1daf445cf493dd018143e9280033f4d83f82b2f)